doc/pingdom.md

```diff
@@ -8,7 +8,7 @@ The IDs of the contacts to be notified are stored as a comma-delimited string in
 
 ### Requirements
 
-* [Terraform](https://www.terraform.io/downloads.html) must be installed. We currently support version `0.6.*`
+* [Terraform](https://www.terraform.io/downloads.html) must be installed. We currently support version `0.7.3`
```

Comment:
I wonder whether it's worth rewording this to refer to the Makefile for the current minimum version.  Otherwise it's likely to get out of date again next time we upgrade.